migrations/db/migrations/20230824080821_dbdev_installer.sql

```diff
@@ -0,0 +1,211 @@
+-- migrate:up
+
+create extension if not exists http with schema extensions;
+create extension if not exists pg_tle;
+
+do $do$
+begin
+    if not exists(select 1 from pgtle.available_extensions() where name = 'supabase-dbdev') then
+
+        perform
+            pgtle.install_extension(
+                'supabase-dbdev',
+                '0.0.2',
+                'Install packages from the database.dev registry',
+                $pkg$
+
+        create schema dbdev;
+
+        create or replace function dbdev.install(package_name text)
+            returns bool
+            language plpgsql
+        as $$
+        declare
+            -- Endpoint
+            base_url text = 'https://api.database.dev/rest/v1/';
+            apikey text = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InhtdXB0cHBsZnZpaWZyYndtbXR2Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2ODAxMDczNzIsImV4cCI6MTk5NTY4MzM3Mn0.z2CN0mvO2No8wSi46Gw59DFGCTJrzM0AQKsu_5k134s';
```

Comment:
This is from a supabase project? And is present in all clients using dbdev?

What's the path forward for if the jwt secret needs to be rolled due to a leak?

Reply:
It's intentional. We'd prefer not to use an apikey at all but afaik that's still not possible.

Good point about if the jwt secret needs to be rolled. I'll bring it up

This PR is not going to be merged b/c it requires adding http and pgtle as default on on new projects so we opted for a template instead for now

closing